dask/array/core.py

```diff
@@ -84,6 +85,15 @@
 )
 from dask.widgets import get_template
 
+if _EMSCRIPTEN:
+    from dask import local
+
+    DEFAULT_GET = local.get_sync
+else:
+    from dask import threaded
+
+    DEFAULT_GET = threaded.get
```

Comment:
It seems odd to have this here. Should we move it to `dask.local` instead?

Reply:
I'm not sure what you mean. Are you objecting to repeating this clause for all the different collections? In principle, it might make sense to consolidate, though I've kept them collection-specific since `Bag` has a different default scheduler.

Reply:
It's just a bit worrying that Emscripten logic is leaking into other parts of the codebase like this.

Think we could do a bit better by containing with some changes in those schedulers and then logic here and elsewhere like

```suggestion
try:
    from dask.threaded import get as DEFAULT_GET
except ImportError:
    from dask.local import get_sync as DEFAULT_GET
```

Maybe this could be handled other ways like having a centralized registry (`dict`?) of different `get`s and we could look for the appropriate one there